packages/neovim/src/utils/findNvim.test.ts

```diff
@@ -119,4 +121,28 @@ describe('findNvim', () => {
       error: undefined,
     });
   });
+
+  it('searches in additional custom paths', () => {
+    const customPaths = ['/custom/path/to/nvim', '/another/custom/path'];
+
+    jest
+      .spyOn(fs, 'existsSync')
+      .mockImplementation(path => customPaths.includes(path as string));
```

Comment:
I'm planning to make findNvim "async", and then this might fail. That's an example of why spies/mocks are brittle.

To test actual behavior, we could specify real paths in the `customPath`, paths that we know exist but aren't valid: e.g. `['README.md', ...]` and then assert the results, which should contain an `invalid` object. 

As a bonus that also allows us to better test the behavior of `invalid`.